catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -8,34 +8,19 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
 
 We will keep this page up to date with the latest information as we make more progress.
 
-## What’s new? (updated 16 August 2019)
+## What’s new? (updated 25 October 2019)
 
-### More of the collection available to search
+### Filtering by date
+We've added the ability to filter search results by their production date and now have production dates for over 90% the catalogue (up from 64% since the middle of August).
 
-We've made more of our digital collections available. You can now find:
-
-- 4337 multi-volume works
-- 900 digitised videos and 3 audio files [(including an 1890 recording of Florence Nightingale)](https://wellcomecollection.org/works/tp9njewm) under an Audio/Video filter.
-
-### Improvements to search relevance
-
-- The way we score a work's relevance to your search query has changed. We now weigh certain fields more heavily like `title`, `subjects`, `genres`, `description`, and `contributors`. If you are searching for a specific title, this should bring that to the top of the results, but also help for generic searches.
-
-Another way we are trying to bring more relevant results to the top of your search is using a strategy called minimum-should-match. This is where we expect a minimum percentage of your terms to be matched in the result.
-
-[You can see the current setup in the code here](https://github.com/wellcometrust/catalogue/blob/d1b4229f6e85c09dd7e5b0c94cffc898d11e23b9/api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkQuery.scala#L15-L28), but this will be evolving over time with your feedback.
-
-### Experimental search features
-
-- We've added some experimental parameters to the API, namely `productionDatesFrom` and `productionDatesTo`. [An example](https://api-stage.wellcomecollection.org/catalogue/v2/works?productionDatesFrom=1900-01-01&productionDatesTo=2000-01-01&query=consumption). We'll be adding some interface elements to this at some stage, and sharing them with you all to get your feedback. So far we have production dates for 64% of our catalogue, and will be adding more soon.
+### Order results
+We've added the ability to order search results by relevance (the default), date ascending and date descending.
 
 ### What are the limits?
```

Comment:
Not sure this heading makes sense without the 'Improvements to search relevance' one that we're deleting. Could maybe change it to something like 'Current search limitations'?